cypress/page/student-registration-form.page.ts

```diff
@@ -0,0 +1,83 @@
+import { data } from "cypress/types/jquery";
+import {PersonalInformation} from "../interfaces/PersonalInformation";
+class StudentRegistrationForm {
+  private nameText: string;
+  private lastNameText: string;
+  private emailText: string;
+  private genderBtn: string;
+  private mobileNumberText: string;
+  private hobbiesCheck: string;
+  private currentAddressText: string;
+  private submitBtn: string;
+  private stateList: string;
+  private cityList: string;
+  private NCROstate: string;
+  private delhiCity: string;
+  private hobbies: string;
+  private myhashhobbies: Map;
+
+  constructor() {
+    this.studentFormPageURL = 'https://demoqa.com/automation-practice-form';
+    this.bodyTr = 'tbody tr';
+    this.nameText = '#firstName'
+    this.lastNameText = '#lastName';
+    this.emailText = '#userEmail';
+    this.genderBtn = '[type="radio"]';
+    this.mobileNumberText = '#userNumber';
+    this.hobbiesCheck = '[type="checkbox"]';
+    this.currentAddressText = '#currentAddress';
+    this.submitBtn = '#submit';
+    this.stateList = '#state > div > div.css-1hwfws3 > div.css-1wa3eu0-placeholder';
+    this.cityList = '#city > div > div.css-1hwfws3 > div.css-1wa3eu0-placeholder';
+    this.NCROstate = "#react-select-3-option-0";
+    this.delhiCity = "#react-select-4-option-0";
```

Comment:
try to find a way to not depend on hard coding the state and the city to select

Reply:
resolved